spec/lib/config_spec.rb

```diff
@@ -157,7 +197,21 @@ module Fusuma
         end
         it 'should return custom interval' do
           action_type = 'swipe'
-          expect(Config.interval(action_type)).to eq 0.3
+          expect(Config.interval(action_type, event_trigger)).to eq 0.3
+        end
+      end
+
+      context 'when interval is set on the trigger' do
+        before do
+          allow(YAML).to receive(:load_file).and_return keymap_with_trigger_interval
```

Comment:
__[RuboCop]__

Metrics/LineLength: Line is too long. [84/80]


<!-- Comment by [Sider](https://sider.review) -->